web-components/src/components/place/PlaceInfo.tsx

```diff
@@ -15,29 +14,27 @@ interface PlaceInfoProps {
   showIcon?: boolean;
 }
 
-interface StyleProps {
-  fontSize?: string;
+interface BodyStyleProps {
   color?: string;
-  smFontSize?: string;
+  fontSizeOverride?: string;
+  smFontSizeOverride?: string;
 }
 
-const useStyles = makeStyles<Theme, StyleProps>((theme: Theme) => ({
-  root: {
-    display: 'flex',
-  },
-  content: props => ({
-    [theme.breakpoints.up('sm')]: {
-      fontSize: props.smFontSize || props.fontSize || theme.spacing(4.5),
+// TODO: this was converted from old MUI makestyles, and I haven't throughly tested the logic works
+const StyledBody = styled(Body, {
+  shouldForwardProp: prop =>
+    prop !== 'fontSizeOverride' && prop !== 'smFontSizeOverride',
+})<BodyStyleProps>(
+  ({ theme, fontSizeOverride, smFontSizeOverride, color }) => ({
+    color: color || theme.palette.primary.contrastText,
+    [theme.breakpoints.up('xs')]: {
+      fontSize: smFontSizeOverride || fontSizeOverride || theme.spacing(3.5),
     },
-    [theme.breakpoints.down('sm')]: {
-      fontSize: props.fontSize || theme.spacing(3.5),
```

Comment:
this component was a bit complex to convert, but I think it was set up incorrectly before my changes (the mobile version would prioritize `fontSize` over `smFontSize`